### ISSUE TITLE
can't get int64 precision output of liteSeg model

### ISSUE THREAD

**Issue by ccqedq** [PaddlePaddle/PaddleSeg#3367]
### 问题确认 Search before asking

- [X] 我已经查询[历史issue](https://github.com/PaddlePaddle/PaddleSeg/issues)(包括open与closed)，没有发现相似的bug。I have searched the [open and closed issues](https://github.com/PaddlePaddle/PaddleSeg/issues) and found no similar bug report.

### Bug描述 Describe the Bug

I want to get int64 precision output of segmentation model
so I change the code in PaddleSeg/tools/export.py from "output_dtype = 'int32' if args.output_op == 'argmax' else 'float32' "
to "output_dtype = 'int64' if args.output_op == 'argmax' else 'float32' ", but I still got the int32 preicision model. and I also want to ask why I can't get float32 precision when there is argmax op existing.

### 复现环境 Environment

-OS： Linux
-PaddleSeg: release/2.8
-Python: 3.8

### Bug描述确认 Bug description confirmation

- [X] 我确认已经提供了Bug复现步骤、代码改动说明、以及环境信息，确认问题是可以复现的。I confirm that the bug replication steps, code change instructions, and environment information have been provided, and the problem can be reproduced.

### 是否愿意提交PR？ Are you willing to submit a PR?

- [X] 我愿意提交PR！I'd like to help by submitting a PR!

**Comment by Asthestarsfalll** [comment]
The output of a segmentation model is a index map, therefore it's unnecessary to use int64, becuase it will not improve the performance of model.

**Comment by Asthestarsfalll** [comment]
`argmax` computes the indices of the max elements of the input tensor’s element along the provided axis，so it should be integer.

**Comment by ccqedq** [comment]
So what precision is the exported onnx model, float32, float64, and what determines this？

**Comment by ccqedq** [comment]
When deploying with rk3588, it is not recommended to export softmax and argmax operators. Why？

**Comment by Asthestarsfalll** [comment]
Internal modules of segmentation model, such as convolution layers, fully connected layers, batch norm......

**Comment by Asthestarsfalll** [comment]
> When deploying with rk3588, it is not recommended to export softmax and argmax operators. Why？

What I mean is keeping int32, instead of discarding argmax.
From the perspective of deploying on edge devices, perhaps these two operators are too resource-intensive.
